src/renderer/ssf-api.ts

```diff
@@ -433,11 +433,12 @@ export class SSFApi {
     /**
      * Opens a modal window to configure notification preference.
      */
-    public showNotificationSettings(): void {
+    public showNotificationSettings(data: string): void {
```

Comment:
@psjostrom I think we discussed about using an object here, any reason. you are using a string? And, the naming convention doesn't match the context either.